sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosItemTest.java

```diff
@@ -151,6 +156,103 @@ public void readItem() throws Exception {
 
     }
 
+    @Test(groups = { "simple" }, timeOut = TIMEOUT)
+    public void readItemWithSoftTimeoutAndFallback() throws Exception {
+        String pk = UUID.randomUUID().toString();
+        String id = UUID.randomUUID().toString();
+        ObjectNode properties = getDocumentDefinition(id, pk);
+        ObjectNode fallBackProperties = getDocumentDefinition("justFallback", "justFallback");
+        container.createItem(properties);
+
+        String successfulResponse = wrapWithSoftTimeoutAndFallback(
+            container
+                .asyncContainer
```

Comment:
We can evaluate - but should chat with the Central Team before spending any time on it. My guess is they will block this.

If Theo adds a public sample it should use CosmosAsyncContainer - just used this here because we don't have a CosmosItemAsyncTest class.

Reply:
Make sense, so only for async API currently